src/serac/physics/solid_mechanics.hpp

```diff
@@ -27,6 +27,16 @@
 
 namespace serac {
 
+/// @brief simple utility for computing matrix norm
```

Comment:
Can we put this somewhere else?  Though not sure if we have a good place for it now.

Reply:
It looks like this isn't being used anywhere.  Maybe we can take it out?